sdk/eventhub/azure-eventhubs/azure/eventhub/aio/client_async.py

```diff
@@ -266,14 +266,16 @@ def create_consumer(
         owner_level = kwargs.get("owner_level")
         operation = kwargs.get("operation")
         prefetch = kwargs.get("prefetch") or self._config.prefetch
+        track_last_enqueued_event_info = kwargs.get("track_last_enqueued_event_info", False)
```

Comment:
Add the docstring for this param

Reply:
docstring has been updated for this param